new/LIT6700Third_Sunday.xml

```diff
@@ -0,0 +1,67 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6700Third_Sunday" xml:lang="en" type="work">
```

Comment:
I woke up now and remembered that we are actually not advised to use any dashes or underscores in record IDs https://betamasaheft.eu/Guidelines/?id=entities-id-structure
I will now approve and merge, but it would be great if you could then rename the record(s) to remove the underscore.
Thank you!